jobs/pywikibot/scripts/MetaMaint.py

```diff
@@ -0,0 +1,24 @@
+# MetaMaint - script to add a random package to MetaKGP homepage.
+
+import pywikibot
+from random import randint
+from datetime import date
+
+def main():
+    site = pywikibot.Site()
+    
+    # Read package blurbs
+    rpage = pywikibot.Page(site, 'Package_blurbs')
```

Comment:
Nit: probably better to put this under the `Metakgp` namespace, like `Metakgp:Package_blurbs`, so it doesn't show up in recent pages, trending, etc.